### ISSUE TITLE
The logo does not have a proper alternative text

### ISSUE THREAD

**Issue by mmiron-HC** [mmiron-HC/WST-RSAMS#1]
User Agent: Chrome 
Language: Both

**Steps to reproduce:** 
from the inspect code - review code behind the image - `<img src="https://articulateusercontent.com/rise/courses/L6Ar3zzW35c_qd5h0eTKMRDl36AC-vvs/NChzF3qfEnM_JWqc.jpg" width="75" height="75" alt="Logo">`
<img src="https://articulateusercontent.com/rise/courses/L6Ar3zzW35c_qd5h0eTKMRDl36AC-vvs/NChzF3qfEnM_JWqc.jpg" width="75" height="75" alt="Logo">

**Expected results:**
Should have a proper alternative text. See Canada.ca/en logo.
